src/noshow/dashboard/layout.py

```diff
@@ -72,11 +68,17 @@ def render_patient_selection(
             )
     with col3:
         st.button(
-            "Volgende patient",
+            "Niet-gebelde patiënt",
```

Comment:
Maybe: "Eerste niet-gebelde patiënt"?
Also maybe we can think of something to make it easier to distinguish between the 4 buttons?